runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -464,6 +467,20 @@ public void onAnimationChanged(AnimationChanged event)
 	{
 		Actor actor = event.getActor();
 
+		if (config.showAbyssalSireStun()
+			&& actor instanceof NPC
+			&& "Abyssal Sire".equals(actor.getName()))
```

Comment:
Use NpcID instead of strings

Reply:
Agreed, now that I think about it, there's no real need to call `removeGameTimer` for this.

Reply:
I'm calling removeGameTimer for the case when the Sire exits the stun phase and leaves the throne.